new-log-viewer/src/typings/worker.ts

```diff
@@ -41,13 +41,15 @@ enum WORKER_REQ_CODE {
     EXPORT_LOG = "exportLog",
     LOAD_FILE = "loadFile",
     LOAD_PAGE = "loadPage",
+    QUERY_LOG = "queryLog",
 }
 
 enum WORKER_RESP_CODE {
     CHUNK_DATA = "chunkData",
     LOG_FILE_INFO = "fileInfo",
-    PAGE_DATA = "pageData",
     NOTIFICATION = "notification",
+    PAGE_DATA = "pageData",
+    CHUNK_RESULT = "chunkResult",
```

Comment:
How about `QUERY_RESULT`?

Reply:
query_results could be a little bit misleading, since it could mean the whole search results. Howevery, this is only a part of the total results